pyanaconda/storage/kickstart.py

```diff
@@ -0,0 +1,199 @@
+#
+# Copyright (C) 2018  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+from blivet.devices import PartitionDevice, TmpFSDevice, LVMLogicalVolumeDevice, \
+    LVMVolumeGroupDevice, MDRaidArrayDevice, BTRFSDevice
+
+from pyanaconda.anaconda_loggers import get_module_logger
+from pyanaconda.core.constants import CLEAR_PARTITIONS_NONE, CLEAR_PARTITIONS_LIST, \
+    CLEAR_PARTITIONS_ALL
+from pyanaconda.modules.common.constants.objects import DISK_SELECTION, AUTO_PARTITIONING, \
+    DISK_INITIALIZATION
+from pyanaconda.modules.common.constants.services import STORAGE
+
+log = get_module_logger(__name__)
+
+__all__ = ["update_storage_ksdata"]
+
+
+def update_storage_ksdata(storage, ksdata):
+    """Update kickstart data to reflect the settings of the storage.
```

Comment:
I suggest:

"Update kickstart data to reflect the current storage configuration."

Reply:
Fixed.